### PR TITLE
[13.0][ADD] website_sale_product_item_cart_custom_qty: New module to allow add to cart custom quantity from product item

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT32818

Please @victoralmau @joao-p-marques ¿can you review it?

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
Wouldn't the plus/minus buttons be beside the add to cart button?

(poor mockup):
image

**Comment by CarlosRoca13** [comment]
> Wouldn't the plus/minus buttons be beside the add to cart button?
> 
> (poor mockup): image

Yes, it could be an option, I will set it as you say

**Comment by CarlosRoca13** [comment]
@chienandalu changes done and test added

**Review by chienandalu — APPROVED** [review]

File: website_sale_product_item_cart_custom_qty/views/assets.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            <script
+                type="text/javascript"
+                src="/website_sale_product_item_cart_custom_qty/static/src/js/add_to_cart.js"
+            />
+            <script
+                type="text/javascript"
+                src="/website_sale_product_item_cart_custom_qty/static/src/js/tour.js"
+            />
+        </xpath>
+    </template>
```
Comment (victoralmau) [inline website_sale_product_item_cart_custom_qty/views/assets.xml]:
```suggestion
    <template id="assets_frontend" inherit_id="website.assets_frontend">
    <xpath expr="." position="inside">
        <script
            type="text/javascript"
            src="/website_sale_product_item_cart_custom_qty/static/src/js/add_to_cart.js"
        />        
    </xpath>
</template>
<template id="assets_tests" inherit_id="website.assets_tests">
    <xpath expr=".">
        <script
            type="text/javascript"
            src="/website_sale_product_item_cart_custom_qty/static/src/js/tour.js"
        />
    </xpath>
</template>
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
Changes done @victoralmau

**Review by victoralmau — CHANGES_REQUESTED** [review]
Sorry but `add_to_cart.js` file is empty. If finally this file is not necessary, please delete the file and what corresponds in `assets.xml` to include it.

**Comment by CarlosRoca13** [comment]
Changed

**Review by victoralmau — APPROVED** [review]
Code review OK.

**Review by pedrobaeza — APPROVED** [review]
Functional review

**Pull request merged**